pkg/http/api.go

```diff
@@ -529,29 +529,29 @@ func (a *api) constructDistributedLockEndpoints() []Endpoint {
 	}
 }
 
-func (a *api) onOutputBindingMessage(reqCtx *fasthttp.RequestCtx) {
-	name := reqCtx.UserValue(nameParam).(string)
-	body := reqCtx.PostBody()
+func (a *api) onOutputBindingMessage(ctx *fasthttp.RequestCtx) {
```

Comment:
Please don't rename `reqCtx` to `ctx`.

Also very important to point out that for as long as we're using fasthttp (hopefully not much longer? See #4979), the context is actually NOT tied to a request, but to the server's own lifecycle (see #4979).

Reply:
Could you describe a bit more what you mean by the context is tied to the server's lifecycle and not the request? Aren't all request contexts tied to both the server _and_ request by virtue of the server's context being wrapped by the request's context?

Reply:
It’s a “feature” of fasthttp (“feature” according to the authors of the library). The context passed to each handler is canceled only when the server is stopped. If the request is canceled by the user, the context is not canceled. I’ve explained it more in #4979, there’s a section about that and a quote from the fasthttp maintainers.